components/SatisfactionSlider.tsx

```diff
@@ -52,6 +66,7 @@ const SatisfactionSlider = ({
           type="range"
           value={number}
         />
+        <output className={styles.invisibleA11yLabel}>{number}</output>
```

Comment:
Correct me if I'm wrong, but it looks like you're trying to use `output` in place of an `aria-live` region here, and first of all it [does not always act that way](https://www.scottohara.me/blog/2019/07/10/the-output-element.html#:~:text=The%20accessibility%20of%20the%20output%20element) (tldr output can work differently on IE browsers). Second of all, I don't think there's a need for either in this case! When a user updates this input, they do not need a live alert because it has not changed anything else on the page. This will essentially just read the users selection out loud to them twice, and I'm not sure that's necessary.

Reply:
Good catch, will remove. I didn't really understand the full usage of `output` when implementing it to be very honest

Reply:
Removed in https://github.com/ibi-group/survey/pull/17/commits/2f7dee15308aeb8cbea934fb3ab8e5529c235cf2